general_qc_report/constants.py

```diff
@@ -128,7 +129,7 @@
     '&status=uploading'
 )
 
-RNA_MAD_QC_FIELDS = [
+RNA_REPLICATION_FIELDS = [
```

Comment:
This should remain same (RNA_MAD_QC_FIELDS) since it is actually fields in quality metric object.